.devauth/config.toml

```diff
@@ -0,0 +1,5 @@
+defaultEnabled = true
+defaultAccount = "main"
+
+[accounts.main]
+type = "microsoft"
```

Comment:
Missing newline.

But does this work? I though DevAuth looked for ~/.devauth, not a .devauth folder in the mod root.

Reply:
Did you forget to add the newline?